example/example-sequelize.js

```diff
@@ -0,0 +1,190 @@
+'use strict'
+
+// Custom Store using Sequelize ORM for PostgreSQL database
+
+// author : https://github.com/the-vishal-kumar/
+// author : https://www.linkedin.com/in/the-vishal-kumar/
+
+// example usage/implementation : https://github.com/the-vishal-kumar/fastify-postgres-boilerplate
```

Comment:
Is this example incomplete without this link?

Reply:
No these are not required. I've just made another commit based on your suggestions.